### ISSUE TITLE
Eyre DNS record is out of date on ~tocwex

### ISSUE THREAD

**Issue by thelifeandtimes** [tocwex/fund#168]
When I hit the "share" button on the ~tocwex/subassembly-pnw campaign from ~sarlev-sarsen's authenticated interface, it copies `http://tocwex.rooftopdao.io/apps/fund/project/~tocwex/subassembly-pnw-` to my clipboard. This is an address that ~tocwex has been accessed from in the past, and technically still can be accessed from, but I don't think it has been accessed from there most recently, or recently at all. How does this link sharing / eyre domain caching actually work?

Maybe we ultimately let the end-user override this in their profile settings somewhere?

**Comment by sidnym-ladrut** [comment]
We're using %eyre scry endpoints for this information. Allowing overrides is a good idea, but will require a state transition to add state indicating that the user has set their own endpoint so %fund shouldn't auto-update using %eyre.

**Comment by thelifeandtimes** [comment]
Yup, I don't think this is hugely urgent, but wanted to log for the future.
